### PR TITLE
add href in link in update message email

### PR DESCRIPTION
**Pull request by aloxe**
### Description:

The link in update email is not clickable. This makes it clickable

### Review

* [ ] Functional review done
* [ ] Usability review done (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] Security review done
* [ ] Code review done
* [ ] Tests were added if useful/possible
* [ ] Reviewed for breaking changes
* [ ] Developer changelog updated if needed
* [ ] Documentation added if needed
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Review by sgiehl — CHANGES_REQUESTED** [review]
@aloxe  Thanks for creating the PR. Seems you actually only changed a test file. So that wouldn't have any effect. To adjust the mails you need to update this file: https://github.com/matomo-org/matomo/blob/4.x-dev/plugins/Marketplace/templates/_updateCommunicationEmail.twig

**Comment by aloxe** [comment]
Hope this is good now.

File: plugins/Marketplace/templates/_updateCommunicationEmail.twig [hunk 1]
```diff
@@ -10,14 +10,14 @@
 {% if hasThemeUpdate %}
 <p>
 {{ 'CoreUpdater_NotificationClickToUpdateThemes'|translate }}<br/>
-{{ host }}index.php?module=CorePluginsAdmin&action=themes
+<a href="{{ host }}index.php?module=CorePluginsAdmin&action=themes">{{ host }}index.php?module=CorePluginsAdmin&action=themes</a>
```
Comment (tsteur) [inline plugins/Marketplace/templates/_updateCommunicationEmail.twig:13]: @aloxe we probably need to escape the first `{{ host }}` like this: `{{ host|e('html_attr') }}`. Also in the link below for better security.

Comment (aloxe) [reply plugins/Marketplace/templates/_updateCommunicationEmail.twig]: sorry for the delay, I haven't noticed this comment. I pushed the change just now.

**Review by tsteur — APPROVED** [review]

**Comment by diosmosis** [comment]
Thanks for the contribution, @aloxe !

**Pull request merged**